thingifier-tests/features/todos/US1_Create_a_todo.feature

```diff
@@ -13,33 +13,32 @@ So that I can remember what assignments and project tasks I need to complete.
 
   Scenario Outline: Student attempts to create a new todo using Json body (Normal Flow)
     When the student sends a POST API request for the "/todos" endpoint with json body containing title "<title>" done status "<doneStatus>" and description "<description>"
-    Then the thingifier app should return a response with status code "201"
-    Then the thingifier app should return a response containing the new todo with title "<title>" done status "<doneStatus>" and description "<description>"
-    Then the thingifier app should contain the new todo with title "<title>" done status "<doneStatus>" and description "<description>"
+    Then the thingifier app should return the error status "201"
```

Comment:
i added a common step def for charles's pull request that is `"the response should have status code {statusCode}"`. This is on main now so u can update it if u like the wording more than this once this is merged